github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -264,32 +264,6 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
-  - always_run: false
-    cluster: prow-workloads
-    decorate: true
-    decoration_config:
-      timeout: 3h0m0s
-    labels:
-      preset-docker-mirror-proxy: "true"
-      preset-podman-in-container-enabled: "true"
-    max_concurrency: 2
-    name: check-provision-k8s-1.25-psa
-    spec:
-      containers:
-      - command:
-        - /usr/local/bin/runner.sh
-        - /bin/sh
-        - -c
-        - cd cluster-provision/k8s/1.25 && KUBEVIRT_PSA='true' ../provision.sh
```

Comment:
Do we need to add `&& KUBEVIRT_PSA='true'` into `check-provision-k8s-1.25`?

Reply:
From a quick look - PSA is only enabled on the 1.26 kubevirt presubmits so I don't think we should enable it for the 1.25 provider check.

Reply:
We don't need to